package.json

```diff
@@ -52,5 +48,6 @@
   },
   "bin": {
     "createDidConfig": "./cli/createDidConfig.js"
-  }
+  },
+  "repository": {}
```

Comment:
what does this do?

Reply:
looks like a debugging artifact: its removed in: [34572c1](https://github.com/KILTprotocol/kilt-extension-api/pull/24/commits/34572c1989d2df3048fb52bf33c45aa221f17f95)